crates/context/src/lib.rs

```diff
@@ -244,6 +245,24 @@ impl ContextManager {
 
         let this = self.clone();
         let context_id = context.id;
+        let _ignored = tokio::spawn(async move {
+            let proxy_contract = this.get_proxy_contract(context_id).await;
```

Comment:
when creating a context, this will always fail, because the context doesn't yet exist on the context config

there's no need for a separate async task here, just call it after `add_context` in the `finalizer` task